### PR TITLE
AP-75 Implement contact us

### PR DESCRIPTION
**Pull request by vwvw**

### CODE REVIEW

File: mobile_app_connector/security/ir.model.access.csv [hunk 1]
```diff
@@ -36,3 +36,4 @@ admin_access_mobile_app_tile_type,admin access_mobile_app_tile_type,model_mobile
 admin_access_mobile_app_tile_subtype,admin access_mobile_app_tile_subtype,model_mobile_app_tile_subtype,sales_team.group_sale_manager,1,1,0,0
 user_access_mobile_app_feedback,User access app feedback,model_mobile_app_feedback,base.group_portal,1,1,1,0
 admin_access_mobile_app_feedback,Admin access app feedback,model_mobile_app_feedback,base.group_system,1,1,1,1
+create_access_mobile_app_public_contact_us,User not logged can create crm claim,model_crm_claim,base.group_public,1,1,1,0
```
Comment (ecino) [inline mobile_app_connector/security/ir.model.access.csv]: If you use sudo() inside the method `mobile_contact_us`, I think it's not needed to add that access rule which would open maybe too much rights

**Review by ecino — CHANGES_REQUESTED** [review]

File: mobile_app_connector/models/request.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        source = parameters["source"]
+        if "contactid" in parameters:
+            contact_id = parameters["contactid"]
+        else:
+            contact_id = None
+
+        claim = self.sudo().create({
+            'email_from': email,
+            'subject': subject,
+            'code': self.env.ref('crm_claim_code.sequence_claim_app').sudo()._next(),
```
Comment (ecino) [inline mobile_app_connector/models/request.py]: Please fix your lines too long.

**Comment by ecino** [comment]
Two flak8 issues:
```
mobile_app_connector/models/request.py:52:70: E502 the backslash is redundant between brackets
mobile_app_connector/models/request.py:53:17: E131 continuation line unaligned for hanging indent
```

**Comment by vwvw** [comment]
Should be good now

File: mobile_app_connector/models/request.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            'subject': subject,
+            'code':
+                self.env.ref('crm_claim_code.sequence_claim_app')
+                    .sudo()._next(),
+            'name': question,
+            'categ_id': self.env['crm.claim.category'].sudo().search(
+                [('name', '=', source)]).id,
+            'partner_id': contact_id,
+        })
+        message = claim.message_post()
```
Comment (ecino) [inline mobile_app_connector/models/request.py]: Why not directly giving the body and subject in the method ? I think the method handles that.
```
        message = claim.message_post(body='...', subject='...')
```

**Review by ecino — APPROVED** [review]

**Pull request merged**